### DISCUSSION TITLE
How do I change my icon theme

### DISCUSSION THREAD

**Discussion by alizehctl** [regolith-linux/regolith-desktop#684] [Q&A] [answered]
I want to override the icon theme for the "BlackHole" regolith look. I remember in Regolith 1.0, I was able to do this by changing this line:

```
#define icon_theme          Arc
```

How would I accomplish the same in Regolith 2.0? I tried adding this to my `~/.config/regolith/Xresources`:

```
gnome.icon.theme: Paper
```

I also tried using the `gnome.icon-theme` variable instead

**Comment by alizehctl** [comment, marked as answer]
Ah never mind, I was able to change the icon theme by using the following steps:

Run `xrdb -query`

```sh
$ xrdb -query | grep icon
gnome.icon-theme:	Paper
gnome.icon.theme:	Paper
gtk.icon_theme_name:	Moka
```

Update `gtk.icon_theme_name` in `~/.config/regolith2/Xresources`:

```
regolith.look.path: /usr/share/regolith-look/blackhole
gnome.icon.theme: Paper
gtk.icon_theme_name: Paper
```

**Comment by kgilmer** [comment]
Yep, glad you got it handled @aadilayub .  `xrdb -query` is a useful tool to look at all settings specified, and then easy to copy/paste into your override file as you've done.
